metricbeat-tests/Makefile

```diff
@@ -7,14 +7,16 @@ QUERY_MAX_ATTEMPTS?=5
 RETRY_TIMEOUT?=3
 STACK_VERSION?=
 METRICBEAT_VERSION?=
+VERSION_VALUE=`cat ../cli/VERSION.txt`
 
 ifneq ($(FEATURE),)
 FEATURE_FLAG=--tags
 endif
 
-.PHONY: build-binary
-build-binary:
-	cd ../cli && go build -o op && mv op ../metricbeat-tests && cd -
+.PHONY: fetch-binary
+fetch-binary:
+	@$(MAKE) -C ../cli build
+	cp ../cli/.github/releases/download/$(VERSION_VALUE)/op .
```

Comment:
This path is not taking into account the arch and OS, so `GOARCH=amd64 make fetch-binary` fails unless the build has been run beforehand without `GOARCH` and `GOOS`. In that case it fetches the binary built with current OS and platform, what usually makes sense, but can lead to unexpected behaviours, like using an older version, or one build in a diferent architecture.

Reply:
Well, I expect using these commands (generation and consumption) in the same platform, so I guess the platform will match with the binary that was created.

The idea of running make this way is avoiding passing GOOS and GOARCH. We only use them on CI side, where we want to produce binaries in all platforms.